lib/rubocop/cop/rspec/factory_bot/factory_class_name.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use string value when setting the class attribute explicitly.
+        #
+        # @example
+        #   # bad
+        #   factory :foo, class: Foo do
+        #   end
+        #
+        #   # good
+        #   factory :foo, class: 'Foo' do
+        #   end
+        class FactoryClassName < Cop
+          MSG = "Pass '%<class_name>s' instead of %<class_name>s."
+
+          def_node_matcher :factory_call?, <<~PATTERN
+            (block (send _ :factory ...) (...) _)
```

Comment:
As far as I remember, a syntax with no block is also possible, e.g.:
```ruby
factory :author, class: User
```
This node pattern will miss it.